docs/setup/configuration.mdx

```diff
@@ -270,6 +270,18 @@ Controls [Django's secret key](https://docs.djangoproject.com/en/3.0/ref/setting
 
 Controls the production assets mount path. Defaults to `/static/`.
 
+### `BREAKER_BOARD_ENABLED`
+
+Enables the [circuit breaker](developer/extending/webhooks/synchronous-events/circuit-breakers.mdx) feature for sync webhooks.
+
+### `BREAKER_BOARD_SYNC_EVENTS`
+
+List of comma separated, lowercase sync webhook events, which should be monitored by the breaker board.
+
+### `BREAKER_BOARD_DRY_RUN_SYNC_EVENTS`
+
+Sets the circuit breaker to dry run mode, in which events will be monitored but no blockage of webhooks will occur when the failure threshold is exceeded.
```

Comment:
What would happen then if apps are not blocked? Does it emit some logs?

Reply:
It does, added that info 👍🏻